packages/@aws-cdk/aws-servicecatalog/test/portfolio.test.ts

```diff
@@ -0,0 +1,216 @@
+import '@aws-cdk/assert-internal/jest';
+import { AccountRootPrincipal, Group, Role, User } from '@aws-cdk/aws-iam';
+import { Stack, Tags } from '@aws-cdk/core';
+import { AcceptLanguage, Portfolio } from '../lib';
```

Comment:
Use the qualified import here too:

```suggestion
import * as servicecatalog from '../lib';
```

This validates everything that is needed is exported by the module.

Reply:
Still not addressed.